tests-e2e/ocw-ci-test-course/homepage.spec.ts

```diff
@@ -72,3 +72,27 @@ test("og:image tag has fully-qualified URL", async ({ page }) => {
     "https://live-qa.ocw.mit.edu/courses/123-ocw-ci-test-course-fall-2022/example_jpg.jpg"
   )
 })
+
+test("twitter:site tag points to correct Twitter account", async ({ page }) => {
```

Comment:
**Comment:** I would be inclined to have a single test, `test("Has expected meta tags in <head>", () => {...})` or something like that.

Playwright is pretty fast, but each of these tests does take ~0.5s on my machine, which adds up eventually. (It's certainly not a problem, but I don't really see the value in separate tests here. If one of them fails, the failing assertion should clearly identify which test failed.)